src/utils/db/car_index/car_index_builder.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use super::{FrameOffset, Hash, IndexHeader, KeyValuePair, Slot};
+use cid::Cid;
+use tokio::io::{AsyncWrite, AsyncWriteExt as _};
+
+#[derive(Debug)]
+pub struct CarIndexBuilder {
+    table: Vec<Slot>,
+    pub longest_distance: u64,
+    pub collisions: u64,
+    capacity: usize,
+}
+
+impl CarIndexBuilder {
+    // Number of buckets given `len` number of elements
+    pub fn capacity_at(len: usize) -> usize {
+        len * 100 / 81
```

Comment:
It's not clear where are those numbers coming from to me.

Reply:
It's the load-factor. I'll add some more documentation.